### PR TITLE
feat(2892331342): Add "mainnet / goerli" dropdown switch to main navbar

### PR DESCRIPTION
**Pull request by yairans-starkware**
### Description of the Changes

* remove goerli text below the starkgate logo 
* add new component (ChainSelect) to switch between mainnet and goerli
* add the env url to the .env files

### Checklist

- [ ] Changes have been done against master branch, and PR does not conflict
- [ ] New unit / functional tests have been added (whenever applicable)
- [ ] Test are passing in local environment
- [ ] Docs have been updated
- [ ] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

---
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/234)

### CODE REVIEW

**Comment by dan-ziv** [comment]
@yairans-starkware please add description and fix linting

**Review by dan-ziv — CHANGES_REQUESTED** [review]

File: src/components/UI/ChainSelect/ChainSelect.module.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import '../../../index';
+
+.chainSelect {
+  margin: 3px 16px;
+}
+
+.selectedIcon {
+  position: absolute;
+  right: 8px;
+}
```
Comment (dan-ziv) [inline src/components/UI/ChainSelect/ChainSelect.module.scss:10]:
```suggestion
.chainSelect {
  margin: 3px 16px;

  .selectedIcon {
    position: absolute;
    right: 8px;
  }
}
```

Comment (yairans-starkware) [reply src/components/UI/ChainSelect/ChainSelect.module.scss:10]: the select items are outside the select itself in the dom tree, so it doesn't work. I would prefer to it also

**Review by dan-ziv — APPROVED** [review]

**Pull request merged**